backup/src/main/java/com/instaclustr/backup/Application.java

```diff
@@ -0,0 +1,63 @@
+package com.instaclustr.backup;
```

Comment:
We probably should merge/refactor this with the operator code at some point, since it uses the same services + servicemanager + guice model

Reply:
See comment on likelihood that backup will be moved into a separate repo